index.js

```diff
@@ -1,39 +1,267 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+/*
+BOILERPLATE STUFF DO NOT TOUCH ++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+const readline = require("readline");
+const readlineInterface = readline.createInterface(
+  process.stdin,
+  process.stdout
+);
 
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     readlineInterface.question(questionText, resolve);
   });
 }
+/*
+END BOILERPLATE STUFF +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
 
-// remember the StateMachine lecture
-// https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
-let states = {
-  'roomOne': { canChangeTo: [ 'roomTwo' ] },
-  'roomTwo': { canChangeTo: [ 'roomThree' ] },
-  'roomThree': { canChangeTo: [ 'roomOne' ] }
-};
-
-let currentState = "green";
-
-function enterState(newState) {
-  let validTransitions = states[currentState].canChangeTo;
-  if (validTransitions.includes(newState)) {
-    currentState = newState;
-  } else {
-    throw 'Invalid state transition attempted - from ' + currentState + ' to ' + newState;
+//DO NOT CHANGE MYFUNK
+myFunk();
+async function myFunk() {
+  //OBJECTS GO HERE!!!!!!!!!!!!!
+  let inv = {
+    //IGNORE FOR NOW
```

Comment:
Is there a reason for inv to be an object and not an array? I find arrays easier to work with for simple collections.